src/Microsoft.IdentityModel.Logging/IdentityModelTelemetryUtil.cs

```diff
@@ -130,6 +130,17 @@ internal static void SetTelemetryData(HttpRequestMessage request)
                 request.Headers.Remove(parameter.Key);
                 request.Headers.Add(parameter.Key, parameter.Value);
             }
+
+            if (additionalHeaders != null)
+            {
+                foreach (var parameter in additionalHeaders)
+                {
+                    // remove this header if it already exists.
+                    // we don't want to add an additional value in case when a telemetry header already exists, but to overwrite it.
+                    request.Headers.Remove(parameter.Key);
```

Comment:
Why would an existing telemetry header exist?

Can we shorten the comment to explain why we need to remove existing?